makefiles/info-nproc.inc.mk

```diff
@@ -0,0 +1,33 @@
+ifneq (, $(filter buildtest info-concurrency, $(MAKECMDGOALS)))
+  ifeq (, $(strip $(NPROC)))
+    # Linux (utility program)
+    NPROC := $(shell nproc 2>/dev/null)
+
+    ifeq (, $(strip $(NPROC)))
```

Comment:
Ifeq automatically strips the arguments IIRC

Reply:
this is just a copy&paste from the old file. please let's merge this as is, I don't have time to work on this atm.

Reply:
I understand, let's get this fix in.